### ISSUE TITLE
Go 101

### ISSUE THREAD

**Issue by RN0311** [asetalias/Logistics-and-Event-Contributions#19]
# Title

**Type**
- [x] Workshop / Hands on
- [ ] Guest Lecture
- [ ] Webinar

**Abstract**

A hands-on tutorial which walks the audience through the first steps of getting started with Golang. I will also, point to further readings to delve deeper into the topics covered during this session.

**Pre-requisites**

- Basic programming concepts
- Operating System basics

**Expected duration**

- 45 mins

**Level**

 - Beginner

**Resources**

 - [Handful of resources](https://github.com/RN0311/WWG-Go-Tutorial-Delhi/blob/master/Resources.md)

**Speaker Bio**

I'm a final year undergrad at IIITD.

**Social Links to your profile**  

GitHub: http://github.com/RN0311
Twitter: https://twitter.com/sparkle_rash

**Comment by RN0311** [comment]
Can this session be in August!?

**Comment by Tanya-Jain** [comment]
Yes, we can plan it in August. We will be sure more on this once the coming up semester starts at the university. Thanks for offering to contribute! We will get back on this soon.

**Comment by Tanya-Jain** [comment]
@RN0311 It would be great if we could have a new possible date for this event to be conducted. Kindly inform if you would be available any other day!

**Comment by RN0311** [comment]
@Tanya-Jain : Yes, any time during weekends in the month of September or October. But, kindly inform at least two weeks prior to this session.

**Comment by Tanya-Jain** [comment]
@RN0311, our events have been restricted to Thursday due to rules laid out by our department, hence limiting our possible days and events. We can have an on-campus event only if you are free on one of the Thursdays of the month, or we can conduct a webinar on any day (even weekend). Kindly update regarding your preference.

**Comment by RN0311** [comment]
@Tanya-Jain: 25th Oct or 1st Nov,(choose any, both works!) will be happy to give hands-on training in Go. as mentioned in the above schedule. :smiley:  I hope it works now? :see_no_evil:
